genomic_data_service/rnaseq/expressions.py

```diff
@@ -0,0 +1,13 @@
+
+
```

Comment:
nit: we can remove these two lines, the pep8 should only complain about the two lines about class definitions if there is something before

Reply:
Agree, fixed.